pages/links.md

```diff
@@ -5,9 +5,10 @@ layout: default
 ---
 
 # Links
-
+ * [Bakersfield - Technologies & Information Meet Ups](https://www.startupdigest.com/digests/bakersfield)
```

Comment:
Agghh, the only time I try to make an edit directly from GitHub and I leave out something.. I knew I should check my edits before.

Any who, I just made the same edit from my desktop without the line and it worked perfectly when I ran the local server test. I although think it'd be wise to just keep the line in there?